hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/V0530TokenSchema.java

```diff
@@ -32,7 +32,7 @@
 
 public class V0530TokenSchema extends StakingInfoManagementSchema {
     private static final Logger logger = LogManager.getLogger(V0530TokenSchema.class);
-    private static final long MAX_PENDING_AIRDROPS = 1_000_000_000L;
+    private static final long MAX_PENDING_AIRDROPS = 20_000_000_000L;
```

Comment:
This is just used in Schema, that defines max keys possible in State. So, should be same as other states in TokenService